bootstrap.go

```diff
@@ -43,6 +43,7 @@ const (
 		Alter_priv   ENUM('N','Y') NOT NULL  DEFAULT 'N',
 		Show_db_priv ENUM('N','Y') NOT NULL  DEFAULT 'N',
 		Execute_priv ENUM('N','Y') NOT NULL  DEFAULT 'N',
+		Index_priv ENUM('N','Y') NOT NULL  DEFAULT 'N',
```

Comment:
This is sql statement text, not golang code. But be alignment is better, I will fix this.